homeassistant/components/minecraft_server/strings.json

```diff
@@ -10,6 +10,9 @@
         }
       }
     },
+    "abort": {
+      "already_configured": "[%key:common::config_flow::abort::already_configured_device%]"
```

Comment:
```suggestion
      "already_configured": "[%key:common::config_flow::abort::already_configured_service%]"
```

Reply:
Why service instead of device? For me device fits better.

Reply:
I think in theory a Minecraft server isn't a device on itself. I would rather call it a service

Reply:
While this is true, in the integrations page it's still called _devices_. That's why I find it a bit inconsistent, but will change it nevertheless.